app/javascript/libs/permissions.js

```diff
@@ -267,6 +267,24 @@ export const SHOW_CHANGE_LOG = [...MANAGE, ACTIONS.CHANGE_LOG];
 
 export const SHOW_FIND_MATCH = [...MANAGE, ACTIONS.READ];
 
-export const VIEW_KPIS = [ACTIONS.READ];
+export const VIEW_KPIS = [
```

Comment:
Could you also add this to `components/permissions/constants.js` I would like to deprecate the file eventually.

Reply:
Sure! I'll maintain the duplicates in the mean time?